src/coreclr/tools/Common/TypeSystem/Interop/IL/Marshaller.cs

```diff
@@ -382,6 +382,50 @@ protected Marshaller()
 
             return marshaller;
         }
+
+        /// <summary>
+        /// Create a marshaller
+        /// </summary>
+        /// <param name="parameterType">type of the parameter to marshal</param>
+        /// <returns>The created Marshaller</returns>
+        public static Marshaller CreateDisabledMarshaller(TypeDesc parameterType,
```

Comment:
There are 3 similar methods:
- `CreateDisabledMarshaller`
- `GetDisabledMarshallingMarshalerType`
- `GetRuntimeDisabledMarshallingMarshallerKind` (in unmanaged runtime)
I would be nice to use the same combination of words in all of them - unify DisabledMarhaller, DisabledMarshalingMarshaler and RuntimeDisabledMarshallingMarshaller to one thing.